support-frontend/assets/helpers/internationalisation/country.ts

```diff
@@ -1,9 +1,9 @@
-import type { $Keys } from 'utility-types';
 // ----- Imports ----- //
+
+import type { $Keys } from 'utility-types';
```

Comment:
We could actually replace wherever this is being used in the file with Typescript's built-in `keyof`- this dependency just helped with moving over from Flow syntax.

Reply:
Just took a look and I think I might make a health card to fix this one up as the types are actually useless atm!

Currently we have an object of type `Record<string, string>` and then we're using `$Keys<typeof object>` which is just `string`. 

It's actually trying to be a union of US states. I reckon we could improve quite a lot here, but probably worth a separate ticket